com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -245,20 +243,61 @@ internal set
 
         private ulong m_LocalClientId;
 
+        internal Dictionary<ulong, NetworkClient> connectedClients = new Dictionary<ulong, NetworkClient>();
```

Comment:
Is there a reason these need to be internal instead of private, given the public getters?

Reply:
For some reason I thought that they had to be internal because I thought we had other classes accessing them via write access (not the IReadOnly property access) but after giving it another look all of the cases where this happens can be removed.

Will refactor this and make them private fields.